force-app/main/default/classes/SOQL.cls

```diff
@@ -345,17 +375,26 @@ public with sharing class SOQL {
      */
     public static void setPrimaryContact() {
         //Use the getOpportunityDetailsForClosedWon() method to get the Opportunities
-
+        List<Opportunity> closedWonOpps = SOQL.getOpportunityDetailsForClosedWon();
```

Comment:
No need to call the class since you're already here, just call the method directly